api/v1/openapi.yaml

```diff
@@ -3501,6 +3501,14 @@ definitions:
       uptime-nanoseconds:
         description: BGP peer connection uptime in nano seconds.
         type: integer
+      graceful-restart:
```

Comment:
Just a personal preference, I would prefer `graceful-restart-enabled` more, also to be consistent with the CRD API.